<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SampleEditor_UI</class>
 <widget class="QDialog" name="SampleEditor_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="targetSampleView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>841</width>
     <height>94</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>CrossCursor</cursorShape>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="ClosePushButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>295</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_1">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>295</y>
     <width>701</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="spinboxesLayout">
    <item>
     <widget class="QLabel" name="label_Start">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="StartFrameSpinBox">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Adjust sample start frame</string>
      </property>
      <property name="maximum">
       <number>1000000000</number>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_Loop">
      <property name="text">
       <string>Loop</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="LoopFrameSpinBox">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Adjust sample loop begin frame</string>
      </property>
      <property name="maximum">
       <number>1000000000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_LoopMode">
      <property name="text">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;Loop&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;mode&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="ProcessingTypeComboBox">
      <property name="minimumSize">
       <size>
        <width>86</width>
        <height>0</height>
       </size>
      </property>
      <property name="toolTip">
       <string>set processing</string>
      </property>
      <item>
       <property name="text">
        <string>forward</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>reverse</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>pingpong</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_LoopCount">
      <property name="text">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;Loop&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;Count&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="LoopCountSpinBox">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>0</height>
       </size>
      </property>
      <property name="toolTip">
       <string>loops</string>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_End">
      <property name="text">
       <string>End</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="EndFrameSpinBox">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Adjust sample end &amp; loop end frame</string>
      </property>
      <property name="maximum">
       <number>1000000000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>838</width>
     <height>267</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="mainSampleViewLayout">
    <item>
     <widget class="QFrame" name="mainSampleview">
      <property name="minimumSize">
       <size>
        <width>624</width>
        <height>265</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>624</width>
        <height>265</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>SplitHCursor</cursorShape>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="mainSampleAdjustView">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>265</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>265</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="verticalzoomSlider">
      <property name="value">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="RubberbandCframe">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>841</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>821</width>
      <height>60</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="rubberbandLabel">
       <property name="text">
        <string>&quot;Rubberband Audio Processor&quot; : Change the tempo (sample length) and pitch of audio.</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Sample length to beat:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="rubberComboBox">
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>off</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1/64</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1/32</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1/16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1/8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1/4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1/2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>13</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>17</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>18</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>21</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>22</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>23</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>24</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>25</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>26</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>27</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>28</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>29</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>32</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ratiolabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pitchlabel">
         <property name="text">
          <string> Pitch (Semitone,Cent)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="pitchdoubleSpinBox">
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Pitch the sample in semitones, cents</string>
         </property>
         <property name="minimum">
          <double>-36.000000000000000</double>
         </property>
         <property name="maximum">
          <double>36.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="r_c_label">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Crispness: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="rubberbandCsettingscomboBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&quot;Crispness&quot; levels:&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  0   equivalent to --no-transients --no-lamination --window-long&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  1   equivalent to --no-transients --no-lamination&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  2   equivalent to --no-transients&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  3   equivalent to --bl-transients&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  4   default processing options&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  5   equivalent to --no-lamination --window-short (may be good for drums)&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;you can find more information here:&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;http://www.breakfastquay.com/rubberband/ &quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;http://www.breakfastquay.com/rubberband/ &lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <item>
            <property name="text">
             <string>0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>841</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="PrevChangesPushButton">
      <property name="text">
       <string>&amp;Apply Changes</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="PlayPushButton">
      <property name="text">
       <string>&amp;Play</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="PlayOrigPushButton">
      <property name="text">
       <string>P&amp;lay original sample</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="newlengthLabel">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>new sample length:</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QComboBox" name="EditTypeComboBox">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="toolTip">
       <string>fade-out type</string>
      </property>
      <item>
       <property name="text">
        <string>volume</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>panorama</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget_4</zorder>
  <zorder>targetSampleView</zorder>
  <zorder>ClosePushButton</zorder>
  <zorder>layoutWidget_1</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>RubberbandCframe</zorder>
 </widget>
 <tabstops>
  <tabstop>ProcessingTypeComboBox</tabstop>
  <tabstop>LoopCountSpinBox</tabstop>
  <tabstop>PrevChangesPushButton</tabstop>
  <tabstop>PlayOrigPushButton</tabstop>
  <tabstop>ClosePushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
